lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -1710,6 +1710,8 @@ void request_initialize(const llvm::json::Object &request) {
   body.try_emplace("supportsLogPoints", true);
   // The debug adapter supports data watchpoints.
   body.try_emplace("supportsDataBreakpoints", true);
+  // Putting version string. Note: this is not part of DAP spec.
+  body.try_emplace("version", g_dap.debugger.GetVersionString());
```

Comment:
please call this `__lldb_version`. We tend to add the `__lldb` prefix to anything that is non-standard.